File: Carro/src/carro/Carro.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package carro;

/**
 *
 * @author usuario
 */
public class Carro {

    String marca; // Atributo
    int placa;    // Atributo

    // Constructor
    public Carro(String marca, int placa) {
        this.marca = marca;
        this.placa = placa;
    }

    // Método
    public void acelerar() {
        int velocidad = 40; // Velocidad inicial

        System.out.println(this.marca + " " + this.placa + " Empezo a acelerar a " + velocidad + " km/h");

        if (velocidad > 50) {
            System.out.println("Multa Excedio el limite de velocidad de 50 km/h");
        }
    }
}

File: Carro.cpp
#include <iostream>
#include <cstdio>

using namespace std;

class Carro {
public:
    string marca;
    int placa;

    Carro(string marca, int placa) {
        this->marca = marca;
        this->placa = placa;
    }

    void acelerar() {
        int velocidad = 100; 

        printf("%s %d Empezo a acelerar a %d km/h\n", this->marca.c_str(), this->placa, velocidad);

        if (velocidad > 50) {
            printf("Multa: Excedio el limite de velocidad de 50 km/h\n");
        }
    }
};

int main() {
    Carro c1("BMW", 907); 

    c1.acelerar();

    return 0;
}
